src/integration-tests/commonTests.js

```diff
@@ -96,6 +96,18 @@ const verifyUserRole = async () => {
 				roleCreationPromises.push(createReviewRole)
 			}
 
+			// Add rollout_manager role creation promise
+			if (existingReviewerRole.statusCode === 400 || !existingReviewerRole.body.result?.data?.length) {
```

Comment:
@adithyadinesh0412  in line no 60 can u fetch the rollout manager role also , if that is not exist then only create, dont merge with existingReviewerRole and also add this in the promise, hence it will parallely run